x-pack/plugins/ml/public/privilege/check_privilege.ts

```diff
@@ -0,0 +1,174 @@
+export function checkPermission(privilegeType: string) {
+  const licenseHasExpired = hasLicenseExpired();
+  return privileges[privilegeType] === true && licenseHasExpired !== true;
+}
+
+// create the text for the button's tooltips if the user's license has
+// expired or if they don't have the privilege to press that button
+export function createPermissionFailureMessage(privilegeType: string) {
+  let message = '';
+  const licenseHasExpired = hasLicenseExpired();
+  if (licenseHasExpired) {
+    message = i18n.translate('xpack.ml.privilege.licenseHasExpiredTooltip', {
+      defaultMessage: 'Your license has expired.',
+    });
+  } else if (privilegeType === 'canCreateJob') {
+    message = i18n.translate('xpack.ml.privilege.noPermission.createMLJobsTooltip', {
+      defaultMessage: 'You do not have permission to create Machine Learning jobs.',
+    });
+  } else if (privilegeType === 'canStartStopDatafeed') {
+    message = i18n.translate('xpack.ml.privilege.noPermission.startOrStopDatafeedsTooltip', {
+      defaultMessage: 'You do not have permission to start or stop datafeeds.',
+    });
+  } else if (privilegeType === 'canUpdateJob') {
+    message = i18n.translate('xpack.ml.privilege.noPermission.editJobsTooltip', {
+      defaultMessage: 'You do not have permission to edit jobs.',
+    });
+  } else if (privilegeType === 'canDeleteJob') {
+    message = i18n.translate('xpack.ml.privilege.noPermission.deleteJobsTooltip', {
+      defaultMessage: 'You do not have permission to delete jobs.',
+    });
+  } else if (privilegeType === 'canCreateCalendar') {
+    message = i18n.translate('xpack.ml.privilege.noPermission.createCalendarsTooltip', {
+      defaultMessage: 'You do not have permission to create calendars.',
+    });
+  } else if (privilegeType === 'canDeleteCalendar') {
+    message = i18n.translate('xpack.ml.privilege.noPermission.deleteCalendarsTooltip', {
+      defaultMessage: 'You do not have permission to delete calendars.',
+    });
+  } else if (privilegeType === 'canForecastJob') {
+    message = i18n.translate('xpack.ml.privilege.noPermission.runForecastsTooltip', {
+      defaultMessage: 'You do not have permission to run forecasts.',
+    });
+  } else if (privilegeType === 'canCreateDataFrameJob') {
+    message = i18n.translate('xpack.ml.privilege.noPermission.createDataFrameJobsTooltip', {
+      defaultMessage: 'You do not have permission to create Data Frame jobs.',
+    });
+  } else if (privilegeType === 'canStartStopDataFrameJob') {
+    message = i18n.translate('xpack.ml.privilege.noPermission.startOrStopDataFrameJobTooltip', {
+      defaultMessage: 'You do not have permission to start or stop Data Frame jobs.',
+    });
+  } else if (privilegeType === 'canDeleteDataFrameJob') {
+    message = i18n.translate('xpack.ml.privilege.noPermission.deleteFrameJobTooltip', {
+      defaultMessage: 'You do not have permission to delete Data Frame jobs.',
```

Comment:
I think sentence case should be used here.

Reply:
Fixed in 5e1ad1c.